raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -185,6 +189,54 @@ describe('Raiden', () => {
     );
   }
 
+  function tokensReducer(state: any, action: RaidenAction): RaidenState {
```

Comment:
This can me merged in the `makeInitialState` function like here: https://github.com/raiden-network/light-client/blob/ac2982ff85ffb2b6efe22b7ad0af8a3b7d5189cc/raiden-ts/tests/unit/raiden.spec.ts#L552

Reply:
I would not like to make it there because this is in response to dispatching `tokenMonitored` action so its better we do it inside the test. `tokenMonitored` action is dispatched in the `monitorToken` function.

Reply:
But you already have a test for `monitorToken`, so no need to test this here again

Reply:
Ok misunderstanding I meant I will use `makeInitialState` inside the test, because there is a `makeinitialState` outside the test where we construct `dummyState`

Reply:
Sure, but this allows you to get rid of the `tokensReducer` completely...